Field/index.jsx

```diff
@@ -225,7 +225,7 @@ const Field = React.createClass({
           borderColor: this.state.hover || focus
             ? customize.borderColorSelected
             : customize.borderColor,
-          boxShadow: focus && `0 0 4px ${customize.borderColorSelected}`
+          boxShadow: focus && (customize.boxShadow ? customize.boxShadow : `0 0 4px ${customize.borderColorSelected}`)
```

Comment:
Just for future reference, as discussed in person, these changes are ok to be carried as this for those customizations won't be officially supported by the Theme/Merchant Theming features.